warehouse/manage/views.py

```diff
@@ -288,6 +300,121 @@ def delete_account(self):
         return logout(self.request)
 
 
+@view_defaults(
+    route_name="manage.account.totp-provision",
+    renderer="manage/account/totp-provision.html",
+    uses_session=True,
+    require_csrf=True,
+    require_methods=False,
+    permission="manage:user",
+)
+class ProvisionTOTPViews:
+    def __init__(self, request):
+        self.request = request
+        self.user_service = request.find_service(IUserService, context=None)
+        self.token_service = request.find_service(ITokenService, name="totp_provision")
+
+    @property
+    def default_response(self):
+        return HTTPSeeOther(self.request.route_path("manage.account"))
+
+    @view_config(request_method="GET")
```

Comment:
We probably want to set this page to no caching ever. e.g. ``Cache-Control: no-store``. I don't think we have a mechanism for doing that currently though, so it might make sense to just raise is as an issue.

Reply:
I'm not extremely familiar with Pyramid, but I *think* we can do this with `@view_config(..., http_cache=0)`. I'll test this tomorrow and see if it sets the headers correctly.

Relevant docs: 

* https://docs.pylonsproject.org/projects/pyramid/en/latest/api/config.html#pyramid.config.Configurator.add_view

* https://docs.pylonsproject.org/projects/pyramid/en/latest/api/view.html#pyramid.view.view_config

Reply:
Confirmed that `http_cache=0` works and adds the relevant response headers. I'll commit this in a moment.